common/src/main/java/org/jboss/hal/testsuite/fragment/formeditor/Editor.java

```diff
@@ -51,6 +53,26 @@ public void text(String identifier, String value) {
     }
 
     /**
+     * Enters text as humanly as possible. E.g. makes pauses between characters and utilizes keyboard shortcuts.
```

Comment:
Please add note that this method should be used only as a workaround in cases when `text(String, String)` is not working properly.